src/ClusterManager/pod_template.py

```diff
@@ -226,8 +226,10 @@ def generate_pods(self, job):
 
         deployment_params = copy.deepcopy(params)
 
-        deployment_params["deployment_replicas"] = params["resourcegpu"]
+        deployment_params["deployment_replicas"] = params["mingpu"]
         deployment_params["LaunchCMD"] = params["cmd"]
+        deployment_params["preemptionAllowed"] = False
+        deployment_params["preemptable"] = "0"
```

Comment:
Please give it a reasonable name maybe `deploymentIndex`, preemptable makes me think the value should be True or False.

Also, please add comment somewhere mentioning why we need two deployments.